src/encode/mod.rs

```diff
@@ -77,6 +79,8 @@ impl<'de> de::Deserialize<'de> for EncoderConfig {
 }
 
 /// A text or background color.
+#[cfg_attr(feature = "config_parsing", derive(serde::Deserialize))]
+#[cfg_attr(feature = "config_parsing", derive(serde::Serialize))]
```

Comment:
can you make these comma-separated rather than separate cfg_attrs?

Reply:
... oh sorry, i didn't realize these pushes were getting populated here. This is still a WIP. i'll make another branch to do my work so this doesn't keep happening.